### PR TITLE
feat(gui): add prerelease versioning

### PR DESCRIPTION
**Pull request by lecoqjacob**
spicy-launcher_iYVW6WEEHa

### CODE REVIEW

**Review by zicklag — APPROVED** [review]
I didn't test, just gave a quick look and nothing stood out as wrong to me.

File: cli/Cargo.toml [hunk 1]
```diff
@@ -1,18 +1,18 @@
 [package]
```
Comment (orhun) [inline cli/Cargo.toml:1]: Please revert the formatting in this file.

File: core/Cargo.toml [hunk 2]
```diff
@@ -1,22 +1,22 @@
 [package]
```
Comment (orhun) [inline core/Cargo.toml:1]: Please revert the formatting in this file.

File: gui/src-tauri/tauri.conf.json [hunk 3]
```diff
@@ -46,8 +46,8 @@
     "windows": [
       {
         "title": "Spicy Launcher",
-        "width": 955,
-        "height": 635,
+        "width": 1280,
+        "height": 720,
```
Comment (orhun) [inline gui/src-tauri/tauri.conf.json:50]: Do we really need to increase the window size of the launcher?

File: gui/src-tauri/src/app.rs [hunk 4]
```diff
@@ -18,6 +18,7 @@ impl App {
         Ok(Self { client, storage })
     }
 
+    /// Returns all versions of the game
```
Comment (orhun) [inline gui/src-tauri/src/app.rs:21]: Let's remove this comment from this file for consistency.

File: gui/package.json [hunk 5]
```diff
@@ -29,8 +35,7 @@
   },
   "dependencies": {
     "@tauri-apps/api": "^1.2.0",
-    "nes.css": "^2.3.0",
-    "node-sass": "^7.0.1",
+    "sass": "^1.58.0",
```
Comment (orhun) [inline gui/package.json:38]: Need to confirm this with @bufgix here.

File: gui/tsconfig.json [hunk 6]
```diff
@@ -1,6 +1,34 @@
...
+    // rollup takes care of these
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "allowSyntheticDefaultImports": true,
+
+    // Hides exports flagged with @internal from the d.ts output
+    "stripInternal": true,
+
+    // TODO: error all the things
```
Comment (orhun) [inline gui/tsconfig.json:25]: Can we resolve this TODO?

File: gui/.eslintrc.js [hunk 7]
```diff
@@ -0,0 +1,57 @@
+module.exports = {
```
Comment (orhun) [inline gui/.eslintrc.js:1]: cc @bufgix

**Review by orhun — CHANGES_REQUESTED** [review]

File: gui/src-tauri/tauri.conf.json (hunk 3 above)
Comment (erlend-sh) [reply gui/src-tauri/tauri.conf.json:50]: I for one would appreciate that change. It feels unnecessarily small on my 13’ MacBook monitor. The changelog text could also use more room for wider text-wrapping.

**Comment by lecoqjacob** [comment]
@orhun should be good for another review

**Review by bufgix — APPROVED** [review]

**Review by orhun — APPROVED** [review]
LGTM

**Pull request merged**